hooks/pre/30-upgrade.rb

```diff
@@ -102,6 +102,25 @@ def mark_qpid_cert_for_update
   end
 end
 
+# rubocop:disable MethodLength
+def recreate_event_queue
+  # piping output to /dev/null because on EL6 any qpid-stat/config commands gives a non fatal python traceback.
+  cert = '/etc/pki/katello/qpid_client_striped.crt'
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" del exchange event --durable > /dev/null 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" add exchange topic event --durable > /dev/null 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" del queue katello_event_queue --force > /dev/null 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" add queue katello_event_queue --durable > /dev/null 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" bind event katello_event_queue entitlement.created 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" bind event katello_event_queue entitlement.deleted 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" bind event katello_event_queue pool.created 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" bind event katello_event_queue pool.deleted 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" bind event katello_event_queue compliance.created 2>&1`
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" queues katello_event_queue > /dev/null 2>&1`
+  if $?.success?
```

Comment:
This only checks the success of the last command, is that intended? If you look at the execute helper, you can see how it collects results from multiple commands.